meson.build

```diff
@@ -2048,13 +2048,12 @@ boot_stubs = []
 
 build_dir_include = include_directories('.')
 
-basic_includes = include_directories(
-        'src/basic',
-        'src/fundamental',
-        'src/systemd',
-        '.')
+basic_includes = [
+        include_directories('src/basic', 'src/fundamental', 'src/systemd', '.'),
+        include_directories('src/basic/linux', is_system : true),
+]
 
-libsystemd_includes = [basic_includes, include_directories(
+libsystemd_includes = basic_includes + [include_directories(
```

Comment:
Also here,
```
libsystemd_includes = [
   basic_includes,
   include_directories(
        ...
   ),
]
```
would be better.

Reply:
I had no clue the lists were flattened, interesting